Helpers/UIStylesHelper.h

```diff
@@ -0,0 +1,18 @@
+//
+//  UIStylesHelper.h
+//  social-maps
+//
+//  Created by César Francisco Barraza on 8/6/18.
+//  Copyright © 2018 Bevin Benson. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
+
+@interface UIStylesHelper : NSObject
++ (void)setRoundedCornersToView:(UIView *)view;
```

Comment:
language nit - i'd make this "setRoundedCornersOnView" or "addRoundedCornersToView" 

similar feedback for the rest :)